.github/ISSUE_TEMPLATE/spec_issue.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Specification Review Issue
+about: Start a request for a specification release
+title: ''
+label: Release Review
+assignees: ''
+---
+
+## Specification issue template
+When creating a specification project lifecycle review, create issues in the  with the content defined as follows.
+- [ ] Specification name and version
+- [ ] Add the label `Release Release`
+- [ ] Naming conventions for artifacts:
+   - [ ] Specification PDF in the form of microprofile-_project_-spec-_version_.pdf
+   - [ ] Specification HTML in the form of microprofile-_project_-spec-_version_.html
+   - _project_ is the microprofile specification short project name (config, health, ...)
+   - _version_ is the two digit x.y version of the specification
+<p>
+
+- [ ] The Nexus Staging links (orgeclipsemicroprofile-NNN where NNN is the staging repository id) which contain all the binaries and relevant documentation:
+   - [ ] [Staging Repo] (add link)
+   - [ ] [Spec PDF] (add link)
+   - [ ] [Spec HTML](add link)
+   - [ ] [Javadoc](add link)
+   - [ ] [API Jar](add link)
+   - [ ] [TCK Jar](add link)
+<p>
+
+- [ ] A link in the Eclipse downloads sections is still available for specs and apidocs:
+   - [ ] https://download.eclipse.org/microprofile/staging/microprofile-_projece_-version/
+   - [ ] An apidocs directory containing the javadoc associated with the API jar.
```

Comment:
FYI, when we do put the promotion scripts in place, the "apidocs" directory can not be promoted.  The scripts only work with files (zips, jars, etc).  Maybe something can be changed down the road, but just letting you know the current limitation.

Reply:
ok, thanks. That will be shame as the javadoc will not be browsable anymore. Maybe we can point to the staging. Anyway, we can figure it out later.